eval/src/vespa/eval/eval/tensor_function.h

```diff
@@ -343,6 +343,9 @@ class Lambda : public Node
 public:
     Lambda(const ValueType &result_type_in, const std::vector<size_t> &bindings_in, const Function &lambda_in, NodeTypes lambda_types_in)
         : Node(result_type_in), _bindings(bindings_in), _lambda(lambda_in.shared_from_this()), _lambda_types(std::move(lambda_types_in)) {}
+    const std::vector<size_t> bindings() const { return _bindings; }
```

Comment:
I think you forgot the & to copying the vector.

Reply:
fixed